src/process.rs

```diff
@@ -129,6 +131,17 @@ impl Process {
             args.extend(&["--headless"]);
         }
 
+        let extension_path = if let Some(load_extension_path) = launch_options.load_extension {
```

Comment:
You could have the `format!` and the `args.extend` all within the one `if let Some(load_extension_path) = …` block, if you wanted, and not worry about making the `extension_path` variable.

Reply:
thank for the feedback, i have made changes as per your feedback :)